lib/Jsrt/ChakraDebug.h

```diff
@@ -22,6 +22,14 @@
 
 #include "ChakraCommon.h"
 
+#if !defined(_WIN32) || !defined(_MSC_VER)
```

Comment:
I would go with `__clang__` instead. For someone not familiar with the intention, it is confusing. I had commented on that usage earlier and the reason given was not having an alternative. IMO `__clang__` is an alternative here.